spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraTableRowReaderProvider.scala

```diff
@@ -64,7 +64,7 @@ trait CassandraTableRowReaderProvider[R] {
               s"but column $columnName is not a regular column in table $keyspaceName.$tableName.")
         case WriteTime(columnName, _) =>
           if (!regularColumnNames.contains(columnName))
-            throw new IOException(s"TTL can be obtained only for regular columns, " +
+            throw new IOException(s"Write time can be obtained only for regular columns, " +
```

Comment:
Looks good but you should probably aim at the earliest branch which is 1.4? or something like that :)

Reply:
@RussellSpitzer pardon me, I couldn't understand what you mean. What should I do?

Reply:
@RussellSpitzer Am I do it write in #1148 PR?